stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/ReactionItemDiffCallback.kt

```diff
@@ -4,10 +4,7 @@ import androidx.recyclerview.widget.DiffUtil
 
 internal class ReactionItemDiffCallback : DiffUtil.ItemCallback<ReactionItem>() {
     override fun areItemsTheSame(oldItem: ReactionItem, newItem: ReactionItem): Boolean {
-        return oldItem.reaction.messageId == newItem.reaction.messageId &&
-            oldItem.reaction.fetchUserId() == newItem.reaction.fetchUserId() &&
-            oldItem.reaction.type == newItem.reaction.type &&
-            oldItem.isMine == newItem.isMine
+        return oldItem.type == newItem.type
```

Comment:
I think it would be great to add a comment explaining why we are relying only on `type`